analyzers/src/SonarAnalyzer.CSharp/Facade/CSharpSyntaxFacade.cs

```diff
@@ -81,5 +82,17 @@ node switch
                 LiteralExpressionSyntax literalExpression => literalExpression.Token.ValueText,
                 _ => string.Empty
             };
+
+        public override SyntaxNode RemoveParentheses(SyntaxNode expression)
```

Comment:
This should just call the `expression.RemoveParentheses()` extension method to avoid duplicate implementation of the same thing.

Reply:
That is fine.